test/unit/services/notification_delivery_service_test.rb

```diff
@@ -37,4 +37,29 @@ def test_invalid_event
       NotificationDeliveryService.new(notification).call
     end
   end
+
+  def test_false_boolean_attribute_is_valid_event
+    account = FactoryGirl.create(:simple_provider, buyer: false)
+    FactoryGirl.create(:active_admin, account: account)
+    event = Accounts::AccountDeletedEvent.create(account)
+    notification = FactoryGirl.build_stubbed(:notification, system_name: :account_deleted)
+    notification.expects(:parent_event).returns(event)
+
+    assert_nothing_raised do
```

Comment:
you can remove `assert_nothing_raised` 
https://github.com/rails/rails/blob/4-2-stable/activesupport/lib/active_support/test_case.rb#L96-L98

Reply:
aaaah sorry, i understood removing the param only 😛 ok 👍